openlibrary/core/sponsorships.py

```diff
@@ -42,7 +42,7 @@ def get_sponsored_editions(user):
     :return: list of editions sponsored by user
     """
     archive_id = get_internet_archive_id(user.key if 'key' in user else user._key)
-    contact_id = get_contact_id_by_username(archive_id)
+    contact_id = get_contact_id_by_username(archive_id) if archive_id else None
```

Comment:
This was needed in order to get sponsorships working locally.  We may want to revert it before merging?